src/sidebar/components/group-list-item.js

```diff
@@ -70,7 +70,7 @@ function GroupListItem({
   /**
    * Opens or closes the submenu.
    *
-   * @param {MouseEvent|KeyboardEvent} event
+   * @param {Event} event
```

Comment:
`toggleSubmenu` is passed as a prop to a component which doesn't specify what kind of `Event` it passes to the callback.